add-on/src/popup/quick-upload.html

```diff
@@ -1,5 +1,4 @@
-<html>
-
+<!DOCTYPE html>
```

Comment:
Adding a doctype is good, but now the `<html>` tag was removed! (And github doesn't escape < html > in review comments... Oops)

Reply:
I think its plain only in review comments. 
Syntax is highlighted correctly when [full file is displayed](https://github.com/ipfs/ipfs-companion/blob/5bee3b0f8df8fe769b537496a0cf14c0ed7e895d/add-on/src/options/options.html).

ps. 🍰